### DISCUSSION TITLE
Potential energy surface calculation

### DISCUSSION THREAD

**Discussion by RicardoCecconello** [ttadano/alamode#234] [Q&A] [answered]
Dear Dr. Tadano,

Given the IFCs calculated with ALAMODE, how could we construct the potential energy surface? Alternatively, given the IFCs and some different structures (different cells and positions), how could we calculate the energy of that structure for a potential generated by the IFCs? I know there is the option of structural optimization on ALAMODE that does something like this, but I would like to maintain the given structure.
What I'd like to do is an analysis like the one in Fig. 1(b) of your paper [Quartic Anharmonicity of Rattlers and Its Effect on Lattice Thermal Conductivity of Clathrates from First Principles](https://journals.aps.org/prl/abstract/10.1103/PhysRevLett.120.105901).

**Comment by ttadano** [comment, marked as answer]
Hi,

Your question may be related to #214.

**Comment by RicardoCecconello** [comment]
Thank you!
